go/chat/storage/inbox.go

```diff
@@ -979,8 +981,30 @@ func (i *Inbox) MembershipUpdate(ctx context.Context, vers chat1.InboxVers, join
 		}
 	}
 	sort.Sort(ByDatabaseOrder(ibox.Conversations))
-	ibox.InboxVersion = vers
 
+	// Update all lists with other people joining and leaving
+	convMap := make(map[string]*chat1.Conversation)
+	for index, c := range ibox.Conversations {
+		convMap[c.GetConvID().String()] = &ibox.Conversations[index]
```

Comment:
It seemed to not be when I was running my tests against it, I started out that way and changed it to this. Isn't `c` a copy of `ibox.Conversations[index]`?